### DISCUSSION TITLE
vercel hosts the site on a different domain

### DISCUSSION THREAD

**Discussion by Aditya99111** [vercel/vercel#7271] [Help] [answered]
first time when i ran vercel it deployed it on https://masters-tau.vercel.app
![image](https://user-images.githubusercontent.com/75898863/148220013-d77cc252-ba33-40c4-a055-3eb0c589351f.png)

when i ran vercel cmmand again it deployed it to https://masters-lopezisaiah7-gmailcom.vercel.app
![image](https://user-images.githubusercontent.com/75898863/148220143-ff8f9160-0681-4c01-bb8d-486f757d3cf4.png)

i have added a custom domain to https://masters-tau.vercel.app and https://masters-lopezisaiah7-gmailcom.vercel.app isnt showing up in my dashboard
any fixes how could i push it to https://masters-tau.vercel.app 
Thanks!

**Comment by leerob** [comment, marked as answer]
The first time you did a production deployment. The next time was a preview deployment. To deploy to production, you need to use `vercel --prod`. 

[Learn more about Automatic URLs](https://vercel.com/docs/concepts/deployments/automatic-urls)
